doc/CONTRIBUTING.md

```diff
@@ -14,3 +14,14 @@ Suggested branches naming convetion: `task_number-task_name`.
 Commit message naming convention: `task_number if implemented this commit will`.
 
 To get task numbers, use: [Trello Card Numbers Plus](https://chrome.google.com/webstore/detail/trello-card-numbers-plus/ncibjlmfhjcjnphnpphgphbflpdpliei/related)
+
+## Documentation
+
+The Swagger (OpenAPI 3) is used for API's documentation.
```

Comment:
Explain in more detail what is documented (the requests), what should be included in a given documentation entry, who should create the documentation (person developing/refactoring given request) and where should it be written (in a yaml file).